applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_hydraulic_solver.py

```diff
@@ -0,0 +1,692 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.kratos_utilities as KratosUtilities
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import base class file
+from KratosMultiphysics.FluidDynamicsApplication.fluid_solver import FluidSolver
+
+
+from pathlib import Path
+
+def CreateSolver(model, custom_settings):
+    return NavierStokesTwoFluidsHydraulicSolver(model, custom_settings)
+
+class NavierStokesTwoFluidsHydraulicSolver(FluidSolver):
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        ##settings string in json format
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "solver_type": "two_fluids_hydraulic",
+            "model_part_name": "",
+            "domain_size": -1,
+            "model_import_settings": {
+                "input_type": "mdpa",
+                "input_filename": "unknown_name",
+                "reorder": false
+            },
+            "material_import_settings": {
+                "materials_filename": ""
+            },
+            "distance_reading_settings"    : {
+                "import_mode"         : "from_mdpa",
+                "distance_file_name"  : "no_distance_file"
+            },
+            "maximum_iterations": 7,
+            "echo_level": 0,
+            "time_order": 2,
```

Comment:
As far as I remember, we only support the alpha scheme in here. If so, I'd remove it.

Reply:
Yes, now it is removed